evalml/tests/component_tests/test_utils.py

```diff
@@ -35,8 +35,12 @@ def test_all_components(
         n_components = 41
     elif is_using_conda:
         n_components = 52
-    elif is_running_py_39_or_above or is_using_windows:
+    elif is_using_windows and not is_running_py_39_or_above:
         n_components = 53
+    elif is_using_windows and is_running_py_39_or_above:
```

Comment:
I'm starting to get confused by all of these +/-1 numbers, lol... Do you think you could add an in-line comment stating which components we expect not to get installed? 🙈

Reply:
Yes!

Reply:
This will be cleaner once sktime supports 3.9 hehe